### PR TITLE
Add missing `blocking` argument documentation to the external source operator

### PR DESCRIPTION
**Pull request by JanuszL**
- as the external source operator documentation is written
  in Python, it missed the `blocking` argument description
  which was defined in the native schema but not copied to
  the python description

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

### Affected modules and functionalities:
- external source operator

### Key points relevant for the review:
- NA

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [x] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Review by jantonguirao — APPROVED** [review]

File: dali/python/nvidia/dali/external_source.py [hunk 1]
```diff
@@ -460,6 +460,11 @@ class ExternalSource:
         .. note::
             This is applicable only when copying data to and from GPU memory.
 
+    `blocking`: bool, optional
+        If True, this operator will block until the data is available
+        (e.g. by calling ``feed_input``).  If False, the operator will raise an error,
+        if the data is not available.
```
Comment (mzient) [inline dali/python/nvidia/dali/external_source.py:466]: I would add some warning that it's intended for advanced usage.
```suggestion
    `blocking`: bool, optional
        **Advanced** if True, this operator will block until the data is available
        (e.g. by calling ``feed_input``).  If False, the operator will raise an error,
        if the data is not available.
```

Comment (JanuszL) [reply dali/python/nvidia/dali/external_source.py]: Fixed here and in the input op.

**Comment by JanuszL** [comment]
!build

**Review by mzient — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

**Pull request merged**